fake-snippets-api/routes/api/orders/create.ts

```diff
@@ -1,36 +1,63 @@
 import { withRouteSpec } from "fake-snippets-api/lib/middleware/with-winter-spec"
 import { z } from "zod"
-import { orderSchema } from "fake-snippets-api/lib/db/schema"
+import { orderSchema, errorSchema, Order } from "fake-snippets-api/lib/db/schema"
+
+const simulate_scenarios = [
+  // TODO: Add more scenarios
+  "gerber-generation-failed",
+  "gerber-upload-failed",
+  "bom-generation-failed",
+  "bom-upload-failed",
+  "pnp-generation-failed",
+  "pnp-upload-failed",
```

Comment:
we should use underscore, this is my mistake, i'm trying to be consistent on the underscore for enum convention but i forget all the time

the reason why: when you make an sdk you can't code enums if there are dashes